pinot-common/src/main/java/org/apache/pinot/common/utils/CommonConstants.java

```diff
@@ -111,6 +111,8 @@ public ServerType getServerType() {
     public static final int DEFAULT_BROKER_QUERY_RESPONSE_LIMIT = Integer.MAX_VALUE;
     public static final String CONFIG_OF_BROKER_QUERY_LOG_LENGTH = "pinot.broker.query.log.length";
     public static final int DEFAULT_BROKER_QUERY_LOG_LENGTH = Integer.MAX_VALUE;
+    public static final String CONFIG_OF_BROKER_QUERY_LOG_MAX_RATE = "pinot.broker.query.log.maxRate";
+    public static final double DEFAULT_BROKER_QUERY_LOG_MAX_RATE = 10_000d;
```

Comment:
Could you indicate the time unit for this? Is it max rate per second, minute?

Reply:
fixed